src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -306,6 +306,7 @@ internal static int GetWarningLevel(ErrorCode code)
                 case ErrorCode.WRN_CallerLineNumberPreferredOverCallerMemberName:
                 case ErrorCode.WRN_CallerLineNumberPreferredOverCallerFilePath:
                 case ErrorCode.WRN_DelaySignButNoKey:
+                case ErrorCode.WRN_AttributeIgnoredWhenPublicSigning:
```

Comment:
Yup, I see the path now. This is fine then.

---
In reply to: [63441401](https://github.com/dotnet/roslyn/pull/11334#discussion_r63441401) [](ancestors = 63441401)